lib/rmt/cli/custom_repos.rb

```diff
@@ -0,0 +1,90 @@
+class RMT::CLI::CustomRepos < RMT::CLI::Base
+
+  include ::RMT::CLI::RepoPrintable
+
+  desc 'add', 'Adds a custom repository to a product'
```

Comment:
Change verb to infinite form: "Add" instead of "Adds" ... same for all descriptions.